notification-hubs-sdk/src/test/java/com/microsoft/windowsazure/messaging/notificationhubs/ConnectionStringTest.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.windowsazure.messaging.notificationhubs;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+
+public class ConnectionStringTest {
+
+    @Test
+    @Ignore
+    public void parse() {
+        Map<String, ConnectionString> testCases = new HashMap<String, ConnectionString>();
+        testCases.put(
+                "Endpoint=sb://marstr-fcm-tutorials.servicebus.windows.net/;SharedAccessKeyName=DefaultListenSharedAccessSignature;SharedAccessKey=9maoDX5iLZqhfK7hhqK5qSIzoY4TecAyAgL0R+w48Gc=",

```

Comment:
We should avoid using your name in the tests.  Let's use something more generic here

Reply:
Is this a valid Hub connection string? For test purposes, I assume we want a valid key value, but we should revoke it on the Hub to make sure it can't be used outside of the tests.

Reply:
D'oh. I've invalidated this access key and updated my local environment.